src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/markup/HtmlStyle.java

```diff
@@ -75,6 +75,16 @@
     typeNameLabel,
     typeNameLink,
 
+    //<editor-fold desc="not-yet-categorized">
+    //
+    // The following constants are to be categorized.
+
+    /**
+     * The class of the {@code pre} element presenting a snippet.
+     */
+    snippet,
+    //</editor-fold>
+
```

Comment:
Suggest removing the editor-fold comments.  It'll be listed as not-yet-categorized by virtue of being in the initial list.

If the code generates standard styles, they should be listed here, so that they get documented in the sometime-forthcoming stylesheet spec.

Reply:
Fixed in 731a1a6be1a.